masterlist.yaml

```diff
@@ -118,6 +118,41 @@ plugins:
       - name: 'Patch for Purists - Floating Rocks.esm'
       - name: 'Patch for Purists - Expansion Implementation.esm'
       - name: 'Patch for Purists - Dialogue Inconsistencies.esm'
+      
+#Other Mods (default)
+  - name: 'Improved Argonians.esp'
+    group: *default
```

Comment:
You don't need to assign the `default` group, all plugins are included in it by default.

Reply:
Thanks!